pymc/distributions/multivariate.py

```diff
@@ -233,6 +233,9 @@ class MvNormal(Continuous):
     def dist(cls, mu, cov=None, tau=None, chol=None, lower=True, **kwargs):
         mu = at.as_tensor_variable(mu)
         cov = quaddist_matrix(cov, chol, tau, lower)
+        # Aesara is stricter about the shape of mu, than PyMC used to be
+        if mu.ndim < 2:
+            mu = at.broadcast_to(mu, cov.shape[0])
```

Comment:
AFAIK, `mu` will be broadcasted to shape (10, 3) in the case below in v3.

```python
>>> pm.MvNormal.dist(mu=np.ones((10, 1)), cov=np.eye(3))
```

More generally, `mu.shape` and `cov.shape[:-1]` should be broadcasted.
Is this case handled here as well?

Reply:
Thanks, let me think about it

Reply:
@Sayam753 I pushed an update which broadcasts `mu` to `cov[..., -1]`